app/assets/images/payment_icons/bri_direct_debit.svg

```diff
@@ -0,0 +1 @@
9.59 0 00-.918 0 .826.826 0 00-.193.556zM24.977 8.797a.43.43 0 01-.127-.315.42.42 0 01.127-.315.43.43 0 01.315-.127.42.42 0 01.315.127.43.43 0 01.127.315c0 .127-.04.228-.127.315a.43.43 0 01-.315.127.43.43 0 01-.315-.127zm.697 3.08h-.764V9.146h.764v2.733zM27.63 9.815c-.229 0-.396.08-.51.24-.114.162-.167.376-.167.637v1.186h-.764V9.145h.764v.362c.1-.114.22-.208.368-.282a.921.921 0 01.449-.12l.007.717h-.148v-.007zM30.409 11.482a1.54 1.54 0 01-1.085.436c-.416 0-.757-.127-1.025-.382-.268-.255-.409-.596-.409-1.025 0-.428.141-.77.416-1.024.274-.255.602-.382.977-.382s.697.114.958.341c.261.228.395.536.395.931v.402h-1.989c.027.148.1.268.235.369a.76.76 0 00.448.14.878.878 0 00.663-.268l.416.462zm-.73-1.62a.589.589 0 00-.382-.134.73.73 0 00-.415.134.524.524 0 00-.221.368h1.198a.57.57 0 00-.18-.368zM32.384 11.241c.255 0 .483-.127.684-.381l.448.509c-.355.368-.73.549-1.145.549-.409 0-.757-.127-1.038-.389-.281-.26-.422-.596-.422-1.018 0-.415.14-.756.429-1.018a1.461 1.461 0 011.018-.395c.194 0 .395.04.596.12.2.081.375.202.529.362l-.395.516a.815.815 0 00-.322-.241.873.873 0 00-.375-.087.76.76 0 00-.516.194c-.14.127-.214.308-.214.543 0 .234.074.415.214.542a.755.755 0 00.51.194zM34.883 9.694v1.22c0 .106.026.187.08.247a.25.25 0 00.194.087c.14 0 .261-.067.362-.194l.281.536c-.234.214-.489.321-.75.321a.92.92 0 01-.663-.261c-.181-.174-.275-.409-.275-.703V9.694h-.321v-.549h.321v-.817h.764v.817h.663v.55h-.657zM22.834 16.405c-.322 0-.61-.134-.864-.408a1.445 1.445 0 01-.382-1.018c0-.409.12-.737.368-.998.248-.262.536-.389.871-.389.335 0 .603.114.81.348v-1.339h.764v3.764h-.763v-.362c-.215.268-.483.402-.804.402zm-.482-1.4c0 .221.067.402.2.543.135.14.289.207.463.207a.57.57 0 00.448-.207.808.808 0 00.188-.543.822.822 0 00-.188-.556.59.59 0 00-.918 0 .826.826 0 00-.193.556zM27.683 15.97a1.54 1.54 0 01-1.085.435c-.415 0-.757-.127-1.025-.382-.268-.254-.408-.596-.408-1.024 0-.429.14-.77.415-1.025.275-.255.603-.382.978-.382s.696.114.957.342c.262.228.395.536.395.93v.41h-1.989c.027.147.1.267.235.368a.76.76 0 00.449.14.878.878 0 00.663-.268l.415.456zm-.73-1.621a.589.589 0 00-.382-.134.73.73 0 00-.415.134.524.524 0 00-.221.368h1.199a.514.514 0 00-.181-.368zM30.235 13.585c.334 0 .622.128.87.389s.369.59.369.998c0 .408-.128.743-.382 1.018-.255.274-.543.408-.864.408-.322 0-.59-.134-.804-.401v.361h-.763v-3.757h.763v1.34c.208-.235.476-.356.81-.356zM29.41 15c0 .22.06.401.187.542a.57.57 0 00.449.208.618.618 0 00.462-.208c.134-.14.201-.321.201-.542a.838.838 0 00-.194-.556.59.59 0 00-.462-.221.577.577 0 00-.456.22.833.833 0 00-.187.557zM32.203 13.284a.43.43 0 01-.127-.315.42.42 0 01.127-.314.43.43 0 01.315-.128c.128 0 .228.04.315.128a.43.43 0 01.127.314.42.42 0 01-.127.315.43.43 0 01-.315.127.405.405 0 01-.315-.127zm.697 3.08h-.763v-2.732h.763v2.733zM34.836 14.181v1.226c0 .107.026.188.08.248a.25.25 0 00.194.087c.14 0 .261-.067.362-.194l.281.536c-.234.214-.489.321-.75.321a.92.92 0 01-.663-.261c-.18-.174-.275-.409-.275-.703v-1.26h-.321v-.549h.321v-.817h.764v.817h.663v.55h-.656z" fill="#F37021"/></svg>
```

Comment:
Can we remove the extra white background (`<path xmlns="http://www.w3.org/2000/svg" fill="#fff" d="M0 0h38v24H0z"/>`)? We're not seeing the grey border

[Video demonstration](https://screenshot.click/24-18-o5kep-pvgd8.mp4)